mungegithub/mungers/testowner/owner.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/gobwas/glob"
```

Comment:
ah, ok, didn't see this Match function, thought I'm gonna use Glob() and would be a hassle. Updated.